sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## 7.5.0-beta.1 (Unreleased)
 
 ### Features Added
+- Move to service API version `2021-05`
```

Comment:
nit: would be nice if everything was in past tense for consistency (moved)

Reply:
ack